conf/openmetadata.yaml

```diff
@@ -241,6 +241,9 @@ secretsManagerConfiguration:
     accessKeyId: ${OM_SM_ACCESS_KEY_ID:-""}
     secretAccessKey: ${OM_SM_ACCESS_KEY:-""}
 
+security:
+  alwaysMaskPasswordsUI: ${ENABLE_MASK_PASSWORDS_UI:-false}
```

Comment:
nit - let's add a comment on the top, something like `This will mask everything even for admins blabla`